pkgs/by-name/se/sessionizer/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  pkgs,
+  buildGoModule,
+}:
+buildGoModule {
+  pname = "sessionizer";
+  version = "0.1.4";
+
+  src = pkgs.fetchFromGitHub {
```

```suggestion
  src = fetchFromGitHub {
```

`fetchFromGitHub` should come from input directly.